src/lib/pages/tx-details/components/decode-message/decode-message-send.tsx

```
@@ -0,0 +1,159 @@
+}) => (
+  <>
+    <Flex>
+      {coins.map((coin) => {
+        const token = coinToTokenWithValue(coin.denom, coin.amount, assetInfos);
+        return (
+          <TokenImageRender
+            alt={getTokenLabel(token.denom, token.symbol)}
+            boxSize={4}
+            logo={token.logo}
+            marginInlineEnd="-4px"
+          />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)